delta/plugins/elasticsearch/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/elasticsearch/AbstractDbSpec.scala

```diff
@@ -0,0 +1,90 @@
+package ch.epfl.bluebrain.nexus.delta.plugins.elasticsearch
+
+import akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit
+import akka.cluster.typed.{Cluster, Join}
+import ch.epfl.bluebrain.nexus.testkit.{IOFixedClock, IOValues, TestHelpers}
+import com.typesafe.config.ConfigFactory
+import monix.bio.Task
+import monix.execution.Scheduler
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+import org.scalatest.{BeforeAndAfterAll, CancelAfterFailure}
+import slick.jdbc.H2Profile.api._
+import slick.jdbc.JdbcBackend
+import slick.jdbc.JdbcBackend.Database
+
+import java.util.UUID
+import scala.concurrent.duration._
+
+abstract class AbstractDbSpec
```

Comment:
Add a `TODO` for this to be moved somewhere else. We don't want to duplicate this class for each module